presto-native-execution/presto_cpp/presto_protocol/presto_protocol.yml

```diff
@@ -192,7 +192,7 @@ JavaClasses:
   - presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TableWriterNode.java
   - presto-main/src/main/java/com/facebook/presto/sql/planner/plan/UnnestNode.java
   - presto-main/src/main/java/com/facebook/presto/sql/planner/plan/WindowNode.java
-  - presto-spi/src/main/java/com/facebook/presto/spi/ErrorType.java
+  - presto-common/src/main/java/com/facebook/presto/common/ErrorType.java
```

Comment:
Move this line closer to ErrorCode.java so that presto-common files stay together.

Reply:
presto-common files are scattered sparsely in the list. I can create another separate PR to sort them out. Otherwise the intended changes won't be obvious in this PR.